docs/user/how-to/write-plans.rst

```diff
@@ -0,0 +1,203 @@
+Writing bluesky plans for blueapi
```

Comment:
Nit: title case and same tense as other pages
```suggestion
Write Bluesky Plans for Blueapi
```

Reply:
I'm always confused by the official style of blueapi/bluesky, but I guess title case takes precedence, otherwise all-lower?

Reply:
Yeah that's exactly how I've treated it